### PR TITLE
DevTools 102 > Elements > Styles > Dark theme shortcut

### PR DESCRIPTION
**Pull request by sofiayem**
Documented https://developer.chrome.com/blog/new-in-devtools-102/#emulation

Merge on May 24.

### CODE REVIEW

File: site/en/docs/devtools/css/reference/index.md [hunk 1]
```diff
@@ -343,6 +343,20 @@ To add a class to an element:
 
 **Figure 14**. The **Element Classes** pane
 
+### Emulate light and dark theme preferences and enable automatic dark mode {: #emulate-light-dark-themes }
+
+To toggle automatic dark mode or emulate the user's preference of light or dark themes:
+
+1. On the **Elements** > **Styles** pane, click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/CNwUGOgogDCqUaQGt9ZS.svg", alt="Toggle common rendering emulations", width="20", height="20" %}**Toggle common rendering emulations**.
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/2G2TqgXtPcUnSgRN4saG.png", alt="Toggle common rendering emulations", width="800", height="376" %}
+1. Select one of the following from the drop-down list:
```
Comment (rachelandrew) [inline site/en/docs/devtools/css/reference/index.md:352]:
```suggestion
1. On the **Elements** > **Styles** pane, click {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/CNwUGOgogDCqUaQGt9ZS.svg", alt="Toggle common rendering emulations.", width="20", height="20" %}**Toggle common rendering emulations**.
   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/2G2TqgXtPcUnSgRN4saG.png", alt="Toggle common rendering emulations.", width="800", height="376" %}
1. Select one of the following from the drop-down list:
```

Comment (rachelandrew) [reply site/en/docs/devtools/css/reference/index.md:352]: Tiny nit but the style guide suggests to use punctuation in alt text https://developers.google.com/style/images?hl=en#alt-text

**Review by rachelandrew — APPROVED** [review]
Just a non-blocking comment re alt text.

File: site/en/docs/devtools/css/reference/index.md [hunk 2]
```diff
@@ -343,6 +343,20 @@ To add a class to an element:
 
 **Figure 14**. The **Element Classes** pane
 
+### Emulate light and dark theme preferences and enable automatic dark mode {: #emulate-light-dark-themes }
+
+To toggle automatic dark mode or emulate the user's preference of light or dark themes:
```
Comment (jecfish) [inline site/en/docs/devtools/css/reference/index.md:348]: "automatic dark mode" link to explainer on what's that

**Review by jecfish — APPROVED** [review]
LGTM % additional link for explaination

**Pull request merged**